caddyhttp/fastcgi/fastcgi_test.go

```diff
@@ -51,6 +52,106 @@ func TestServeHTTP(t *testing.T) {
 	}
 }
 
+// listenerWithConnectionCounter adds a counter to keep track
+// of the number of accepted connections.
+type listenerWithConnectionCounter struct {
```

Comment:
It is not in any way less descriptive but rather makes the code more readable. Long names should be means of distinction which is not the case here.

Plus, its the Go way to have short names https://talks.golang.org/2014/names.slide#7.